jenkins_jobs/build_pipelines.groovy

```diff
@@ -0,0 +1,29 @@
+def createPipeline(String repositoryName) {
+  multibranchPipelineJob(repositoryName) {
+    branchSources {
+      github {
+        scanCredentialsId('github-user-pass-credentials')
+        repoOwner(GITHUB_ORGANIZATION_NAME)
+        repository(repositoryName)
+        buildForkPRHead(true)
+        buildForkPRMerge(false)
+        buildOriginBranch(false)
+        buildOriginBranchWithPR(false)
+        buildOriginPRHead(true)
+        buildOriginPRMerge(false)
+      }
+    }
+    triggers {
+      periodic(1)
```

Comment:
This has a periodic trigger but `createPeriodicPipeline` don't, is that correct?